types/hapi/index.d.ts

```diff
@@ -1938,7 +1938,7 @@ export class Request extends Podium {
      * an object containing the query parameters.
      * TODO update with outcome of: https://github.com/hapijs/hapi/pull/3479
      */
-    query: Object;
+    query: any;
```

Comment:
No problem, but how to run tests of types/hapi? I cannot run existed tests even though I followed the guide on README of DefinitelyTyped.